Tests/swift-sdk-swift-tests/IterableAPITests.swift

```diff
@@ -589,24 +590,20 @@ class IterableAPITests: XCTestCase {
         let config = IterableConfig()
         IterableAPI.initializeForTesting(apiKey: IterableAPITests.apiKey, config: config, networkSession: networkSession)
         IterableAPI.email = "user@example.com"
-        IterableAPI.get(
-            inAppMessages: 1,
-            onSuccess: {(_) in
-                let expectedQueryParams = [
-                    (name: AnyHashable.ITBL_KEY_API_KEY, value: IterableAPITests.apiKey),
-                    (name: AnyHashable.ITBL_KEY_COUNT, value: 1.description),
-                    (name: AnyHashable.ITBL_KEY_PLATFORM, value: .ITBL_PLATFORM_IOS),
-                    (name: AnyHashable.ITBL_KEY_SDK_VERSION, value: IterableAPI.sdkVersion),
-                    ]
-                TestUtils.validate(request: networkSession.request!,
-                                   requestType: .get,
-                                   apiEndPoint: .ITBL_ENDPOINT_API,
-                                   path: .ITBL_PATH_GET_INAPP_MESSAGES,
-                                   queryParams: expectedQueryParams)
-                expectation1.fulfill()
-        },
-            onFailure:nil
-        )
+        
+        IterableAPI.get(inAppMessages: 1,
+                        onSuccess: { (_) in
+                            let expectedQueryParams = [(name: AnyHashable.ITBL_KEY_API_KEY, value: IterableAPITests.apiKey),
+                                                       (name: AnyHashable.ITBL_KEY_COUNT, value: 1.description),
+                                                       (name: AnyHashable.ITBL_KEY_PLATFORM, value: .ITBL_PLATFORM_IOS),
+                                                       (name: AnyHashable.ITBL_KEY_SDK_VERSION, value: IterableAPI.sdkVersion)]
+                            TestUtils.validate(request: networkSession.request!,
+                                               requestType: .get,
+                                               apiEndPoint: .ITBL_ENDPOINT_API,
+                                               path: .ITBL_PATH_GET_INAPP_MESSAGES,
+                                               queryParams: expectedQueryParams)
+                            expectation1.fulfill()},
```

Comment:
nit: `}` isn't very visible here, I think placing the closing bracket on a new line looks better.